.lighthouse/jenkins-x/release/promote-vs.sh

```diff
@@ -11,7 +11,7 @@ declare -a repos=(
   # GKE
   "jx3-gke-vault" "jx3-gke-gsm" "jx3-gke-gcloud-vault" 
   # EKS
-  "jx3-eks-terraform-vault"
+  "jx3-eks-terraform-vault" "jx3-eks-vault"
```

Comment:
We can remove `jx3-eks-terraform-vault` :+1:

Reply:
the BDD tests have already triggered so may as well leave this and remove in a followup PR

Reply:
sgtm